walkthroughs/week-3-server/examples/todo-list/src/main/java/com/google/sps/servlets/DeleteTaskServlet.java

```diff
@@ -31,7 +31,6 @@
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
What are the tasks? Do we support anything related to tasks?

Reply:
Oops this is an unrelated example in another directory